src/main/java/com/example/bookstoreapp/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package com.example.bookstoreapp.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
using records for dto is better practice. Let’s  use it for  each dto if it is possible